src/shared/api/server/token-fetch.tsx

```diff
@@ -8,7 +8,8 @@ const getRegistry = (chainId: string): Registry => {
   return registryClient.bundled.get(chainId);
 };
 
-export const fetchAllTokenAssets = (chainId: string): Token[] => {
+// TODO: Deprecated. Use remote ChainRegistryClient directly.
+export const fetchAllTokenAssets_deprecated = (chainId: string): Token[] => {
```

Comment:
Want to ensure no one uses this as it creates its own `Token` type primitive and uses bundled registry vs remote registry client.